modules/python/python-logmsg.c

```diff
@@ -49,6 +49,15 @@ py_log_message_getattr(PyLogMessage *self, gchar *name)
 }
 
 static void
+py_log_message_setattr(PyLogMessage *self, gchar *name, gchar *value)
+{
+  NVHandle handle;
+
+  handle = log_msg_get_value_handle(name);
+  log_msg_set_value(self->msg, handle, value, -1);
+}
+
```

Comment:
@bazsi: for those identifiers `setattr(logmsg, key, value)` also works (but we can support that ... )